gerrit/install.sls

```diff
@@ -24,12 +24,36 @@ group_{{ settings.group }}:
     - group: {{ settings.group }}
     - makedirs: true
 
+{{ settings.base_directory }}/{{ settings.site_directory }}/lib:
+  file.directory:
+    - user: {{ settings.user }}
+    - group: {{ settings.group }}
+    - makedirs: true
+
+{% for name, library in salt['pillar.get']('gerrit:libraries', {}).items() %}
+{{ install_dir }}/{{ site_dir }}/lib/{{ name }}.jar:
+  file.managed:
+    - source: {{ library.source }}
+    - source_hash: {{ library.source_hash }}
+    - user: {{ user }}
+    - group: {{ group }}
+{% endfor %}
+
 {{ settings.base_directory }}/{{ settings.site_directory }}/plugins:
   file.directory:
     - user: {{ settings.user }}
     - group: {{ settings.group }}
     - makedirs: true
 
+{% for name, plugin in salt['pillar.get']('gerrit:plugins', {}).items() %}
```

Comment:
Agreed. The README.md has always implied that a person could opt to reference the pillar.example and to date on has not existed. I’ll get one whipped up and incorporated in to this pull request.

Thanks Forrest!

> On Mar 21, 2016, at 3:14 PM, Forrest <notifications@github.com> wrote:
> 
> In gerrit/install.sls <https://github.com/saltstack-formulas/gerrit-formula/pull/4#discussion_r56879817>:
> 
> >  {{ settings.base_directory }}/{{ settings.site_directory }}/plugins:
> >    file.directory:
> >      - user: {{ settings.user }}
> >      - group: {{ settings.group }}
> >      - makedirs: true
> >  
> > +{% for name, plugin in salt['pillar.get']('gerrit:plugins', {}).items() %}
> There's no pillar file for the gerrit formula and I don't see the plugins defined anywhere else other than the directory creation. Could you please add a pillar.example that provides users with an example of specifying their plugins?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/saltstack-formulas/gerrit-formula/pull/4/files/6c5012e5fc99f0a41d5d3fea5b51c9681d9a8762#r56879817>